configs/live/canto/nfts.json

```diff
@@ -0,0 +1,8 @@
+{
+    "output": "generated/live/canto/nfts.",
+    "chain": "canto",
```

Comment:
What chain is that ? IMO we should stick to chains supported by the decentralized network.